utils/handlers/functions.go

```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/spaceuptech/space-cloud/modules/functions"
 )
 
-// HandleRequest creates a Functions request endpoint
-func HandleRequest(functions *functions.Module, auth *auth.Module) http.HandlerFunc {
+// HandleFunctionCall creates a Functions request endpoint
+func HandleFunctionCall(functions *functions.Module, auth *auth.Module) http.HandlerFunc {
```

Comment:
In my opinion, should be called `HandleFunctions` or`HandleCustomFunctions`. Easier to understand. What do you think?

Reply:
This is the handler invoked when the `api.call()` method is called on the client. Technically it is handling a _function call_ event right? Hence the name HandleFunctionCall.
Makes sense?

Reply:
Ahh makes sense. Yeah, everything else seemed good.